### PR TITLE
jq: add German translation

### PR DESCRIPTION
**Pull request by Lodifice**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/jq.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# jq
+
+> Ein JSON-Verarbeiter für die Kommandozeile mit einer domänenspezifischen Sprache.
+> Weitere Informationen: <https://stedolan.github.io/jq/manual/>.
+
+- Führe den angegebenen Ausdruck aus (gib farbiges und formatiertes JSON aus):
+
+`{{cat pfad/zu/datei.json}} | jq '{{.}}'`
```
Comment (EmilySeville7cfg) [inline pages.de/common/jq.md:8]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/jq.md) page should be change before this fix is applied here:
```suggestion
`{{cat pfad/zu/datei.json}} | jq '.'`
```

Optional change: our style guide doesn't handle such cases.

File: pages.de/common/jq.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`{{cat pfad/zu/datei.json}} | jq '{{.[index1], .[index2], ...}}'`
+
+- Gib alle Listenelemente/Objektschlüssel aus:
+
+`{{cat pfad/zu/datei.json}} | jq '.[]'`
+
+- Füge bestimmte Schlüssel hinzu/lösche bestimmte Schlüssel:
+
+`{{cat pfad/zu/datei.json}} | jq '{{.}} {{+|-}} {{{"schlüssel1": "wert1", "schlüssel2": "wert2", ...}}}'`
```
Comment (EmilySeville7cfg) [inline pages.de/common/jq.md:32]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/jq.md) page should be change before this fix is applied here:

```suggestion
`{{cat pfad/zu/datei.json}} | jq '. {{+|-}} {{{"schlüssel1": "wert1", "schlüssel2": "wert2", ...}}}'`
```

Optional change: our style guide doesn't handle such cases.

**Review by EmilySeville7cfg — APPROVED** [review]
- PR was approved because all found fixes are optional.
- Thanks for the contribution. ❤️

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages.de/common/jq.md (hunk 1 above)
Comment (pixelcmtd) [reply pages.de/common/jq.md:8]: If you think that this should be changed, please open a separate PR changing it in all languages.

**Pull request merged**